src/nodes/core/UniformGroupNode.js

```diff
@@ -8,22 +21,49 @@ class UniformGroupNode extends Node {
 
 	}
 
+	/**
+	 * Constructs a new uniform group node.
+	 *
+	 * @param {String} name - The name of the uniform group node.
+	 * @param {Boolean} [shared=false] - Whether this uniform group node is shared or not.
+	 * @param {Number} [order=1] - Influences the internal sorting.
+	 */
 	constructor( name, shared = false, order = 1 ) {
 
 		super( 'string' );
 
+		/**
+		 * The name of the uniform group node.
+		 *
+		 * @type {String}
+		 */
 		this.name = name;
-		this.version = 0;
```

Comment:
FYI: `version` and `needsUpdate` were defines in this class although it is derived from `Node`. I've removed the redefinition since it is equal to `Node.version` and `Node.needsUpdate`.